cmd/cli/kubectl-kyverno/log/log.go

```diff
@@ -33,3 +41,26 @@ func isVerbose(args ...string) bool {
 	}
 	return false
 }
+
+func getLogLevel(args ...string) (int, error) {
```

Comment:
This library uses the `github.com/spf13/cobra` library, so instead of parsing the string argument directly, you should use the reference value Cobra provides to this particular arg. 

This way you would be able to have the default value set there, and avoid needing all this code the loop through strings and parse and the manner in which you're doing the "default" value in the code above.

Reply:
We are using `klog` to set all the logging-related flags
I think we are not doing this because we are reading the value of `v` in the root cmd, and `flag.Lookup` returns the correct value at root cmd but returns 0 for subcommands `apply`, `test`. That might be the reason why we went with this in the first place